### PR TITLE
Vinberg overhaul

### PR DESCRIPTION
**Pull request by HechtiDerLachs**
This contains the changes from #2665 and #2681 and should hence be rebased after the respective merging. 

On top of it, it contains updates to the elliptic surfaces and some other fixes that allow us to significantly shorten our script for our Oscar book chapter.

### CODE REVIEW

**Comment by simonbrandhorst** [comment]
Can be rebased now.

**Comment by HechtiDerLachs** [comment]
I was afraid a rebase would be too tedious. Since everything will be squashed, eventually, I just merged.

File: experimental/Schemes/BlowupMorphism.jl [hunk 1]
```diff
@@ -617,20 +617,3 @@ function pushforward(f::BlowupMorphism, g::VarietyFunctionFieldElem)
   pfg = fraction(pullback(phi)(OO(V)(numerator(h))))//fraction(pullback(phi)(OO(V)(denominator(h))))
   return FY.(pfg)
 end
-
-@attr AbsSpecMor function isomorphism_on_open_subset(phi::AbsCoveredSchemeMorphism)
```
Comment (simonbrandhorst) [inline experimental/Schemes/BlowupMorphism.jl]: Why is this and the following removed? It is called later on in line 496

Comment (HechtiDerLachs) [reply experimental/Schemes/BlowupMorphism.jl]: Good catch! Looks like a merge gone wrong.

File: experimental/Schemes/elliptic_surface.jl [hunk 2]
```diff
@@ -511,18 +515,22 @@ function relatively_minimal_model(E::EllipticSurface)
     Y0 = Y1
     inc_Y0 = inc_Y1
     X0 = X1
+    set_attribute!(Y0, :is_irreducible, true)
+    set_attribute!(Y0, :is_reduced=>true)
```
Comment (fingolfin) [inline experimental/Schemes/elliptic_surface.jl]: I am impartial to which syntax variant you use for setting attributes, but it is a bit jarring that in this code block it alternates between the two repeatedly ;-).

Feel free to leave it and ignore me :-)

File: experimental/Schemes/elliptic_surface.jl [hunk 3]
```diff
@@ -511,18 +515,22 @@ function relatively_minimal_model(E::EllipticSurface)
     Y0 = Y1
     inc_Y0 = inc_Y1
     X0 = X1
+    set_attribute!(Y0, :is_irreducible, true)
```
Comment (simonbrandhorst) [inline experimental/Schemes/elliptic_surface.jl]:
```suggestion
    set_attribute!(Y0, :is_irreducible=> true)
```

File: experimental/Schemes/elliptic_surface.jl [hunk 4]
```diff
@@ -511,18 +515,22 @@ function relatively_minimal_model(E::EllipticSurface)
...
+    set_attribute!(Y0, :is_reduced=>true)
+    set_attribute!(Y0, :is_integral=>true)
+    set_attribute!(X0, :is_irreducible, true)
```
Comment (simonbrandhorst) [inline experimental/Schemes/elliptic_surface.jl]:
```suggestion
    set_attribute!(X0, :is_irreducible=> true)
```

**Review by simonbrandhorst — APPROVED** [review]

File: experimental/Schemes/elliptic_surface.jl (hunk 2 above)
Comment (simonbrandhorst) [reply experimental/Schemes/elliptic_surface.jl]: Agreed.

**Pull request merged**